test/fail_compilation/retscope.d

```diff
@@ -626,3 +626,15 @@ struct Result(R)
     n.empty();
 }
 
+/**********************************************/
```

Comment:
Yeah then put it in `test/runnable/testscope.d`, which is for test that are passing.

It's weird to have a test that is supposed to compile in a file that test things that should be rejected.